src/Entity/OfferingCollection.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace SandwaveIo\Acronis\Entity;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class OfferingCollection
+{
+    /**
+     * @var Offering[]
+     * @Serializer\Type("array<SandwaveIo\Acronis\Entity\Offering>")
+     * @Serializer\SerializedName("items")
+     */
+    private $offeringItems;
+
+    /**
+     * @var Offering[]
+     * @Serializer\Type("array<SandwaveIo\Acronis\Entity\Offering>")
+     * @Serializer\SerializedName("offering_items")
+     * @Serializer\Groups({"update_data"})
+     */
+    private $updatedOfferingItems;
+
+    public function __construct()
+    {
+        $this->offeringItems = [];
+        $this->updatedOfferingItems = [];
+    }
+
+    public function getOfferingItems(): array
```

Comment:
voor de getters en settings zou ik in dit geval wel een docblock toevoegen, omdat je array type gebruikt als parameter voor setters en als return type voor getters
in docblock can je dan aangeven dat de getters/setters bestaat uit een array van Offering entities